chrome-headless/Dockerfile

```diff
@@ -1,130 +1,13 @@
-FROM openjdk:10-jdk-slim
-LABEL version="1.0" maintainer="Kayla Altepeter"
-# ENV CHROME_VERSION 72.0.3626.81
-ENV SONAR_SCANNER_VERSION 3.2.0.1227
-ENV NODE_VERSION 9.11.2
-ENV YARN_VERSION 1.7.0
-ENV CHROME_DRIVER_VERSION 2.46
+FROM node:12.14.0-buster-slim
+LABEL version="1.1" maintainer="Kayla Altepeter"
 
-# install tools
-# https://github.com/docker-library/buildpack-deps/blob/b0fc01aa5e3aed6820d8fed6f3301e0542fbeb36/sid/curl/Dockerfile
-# plus git and ssh
-RUN apt-get update && apt-get install -y --no-install-recommends \
-    ca-certificates \
-    curl \
-    netbase \
-    wget \
-    vim \
-    openssh-client \
-    git \
-    python3 \
-    jq \
```

Comment:
I need to keep these extra tools for ci processes that use them

Reply:
most of these come in the base node image so they don't need to be explicitly installed